Fireworks/Eve/plugins/DummyEvelyser.cc

```diff
@@ -205,41 +208,49 @@ void DummyEvelyser::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
 {
    printf("DummyEvelyser::analyze\n");
 
+  edm::Handle<reco::TrackCollection> trackHandle;
+   iEvent.getByToken(trackCollectionToken_, trackHandle);
+   const reco::TrackCollection trackCollection = *(trackHandle.product());
+   if (!trackHandle.isValid()) {
+      edm::LogError("DummyEvelyser")
+         << "Error! Can't get Track collection "
+         << std::endl;
+      return;
+   }
+   
    if (m_eve)
    {
       // Remake geometry if it has changed.
       m_geomWatcher.check(iSetup);
 
-
       // Stripped down demo from Tracking twiki.
-
+ 
       using namespace edm;
 
-      Handle<View<reco::Track> >  tracks;
-      iEvent.getByLabel(m_trackTags, tracks);
-
       m_trackList->DestroyElements();
 
       // All top-level elements are removed from default event-store at
       // the end of each event.
       m_eve->AddElement(m_trackList);
 
       int cnt = 0;
-      for (View<reco::Track>::const_iterator itTrack = tracks->begin();
-           itTrack != tracks->end(); ++itTrack, ++cnt)
+      
+      for (auto itTrack = trackCollection.begin();
+           itTrack != trackCollection.end(); ++itTrack, ++cnt)
       {
          TEveTrack* trk = fireworks::prepareTrack(*itTrack, m_trackList->GetPropagator());
          trk->SetElementName (TString::Format("Track %d", cnt));
          trk->SetElementTitle(TString::Format("Track %d, pt=%.3f", cnt, itTrack->pt()));
          trk->MakeTrack();
+         
          trk->SetAttLineAttMarker(m_trackList);
          m_trackList->AddElement(trk);
-
+   
          // The display() function runs the GUI event-loop and shows
          // whatever has been registered so far to eve.
          // It returns when user presses the "Step" button (or "Continue" or
          // "Next Event").
-         m_eve->display(std::string("DummyEvelyser::analyze done for:\n") + trk->GetName());
+         //m_eve->display(std::string("DummyEvelyser::analyze done for:\n") + trk->GetName());
```

Comment:
@alja for my understanding, what is the implication of this comment? From the comment in the code I would say that a functionality is suppressed, and the loop is just use to update m_tracklist